### DISCUSSION TITLE
Great 0.37.0 release

### DISCUSSION THREAD

**Discussion by minthemiddle** [notaapp/talk#250] [Random]
Version 0.37.0 is another great one, congrats guys 🎉!

I just played around with Logseq's daily note functionality and the [Raycast extension](https://www.raycast.com/futantan/logseq) that adds a global keyboard shortcut to add entries to the daily log quickly for the past days. I much prefer to use Nota and am happy about the new daily note feature and how it will evolve.

**Comment by astoilkov** [comment]
☺️ Ooo, thanks.

The next release will include improvements to daily notes together with configuration options. We like how we were able to integrate daily notes into the search. We want to make it even more integrated.

As daily notes are still an experiment, if you have something you wish for, you can share it.

**Reply by webwesen** [reply]
configurable rendered titles in daily notes would be nice .
I would prefer keeping yyyy-mm-dd (2022-06-13) rather than 'Mon, 13 Jun'

**Reply by astoilkov** [reply]
Great. This is what we are thinking as well.
